### DISCUSSION TITLE
另一则有趣的测试盘面，看棋软也会举棋不定或是迷之自信？

### DISCUSSION THREAD

**Discussion by ATH139** [official-pikafish/Pikafish#59] [General]
3ak1b2/4ar3/2R3nr1/p1p1p3p/6p2/2PN5/P3P3P/4C1N2/C3A4/2BAK3c r

说它有趣，实则是因为它可以对引擎的自信力做一个不小的考验！
如果您仅以引擎是否快速给出正解来下某种结论的话，那么真的单纯了~
这个盘面要看的是：
1、引擎是否能快速找出正解：即‘**炮五平八**’！（以皮卡鱼为例，可以秒解）
2、即便在找到了正招后，引擎是否还在‘炮九平八’和‘炮五平八’之间犹豫不决？
（多等几秒或几十秒看看，强机计算优势此时凸显）
3、部分棋软在坚持‘炮九平八’的错误路线下，引擎是否认为局面是红亏？（主要看
审局思维及打分准确度）

以上三点才是我们本次测试手中引擎的主要目的！（关闭云库）

**Comment by vocanet** [comment]
218稳定的解决
